apps/src/blockly/addons/cdoUtils.js

```diff
@@ -91,3 +92,13 @@ export function getField(type) {
   }
   return field;
 }
+
+/**
+ * Returns a theme object, based on the presence of an option in the browser's localStorage.
+ * @param {string} type
+ * @returns {?Blockly.Field}
+ */
+// Users can change their active theme using the context menu. Use this setting, if present.
+export function chooseTheme(themeOption) {
```

Comment:
nit: would a better name for this function be `getTheme`?

Reply:
`getTheme` is a core Blockly function that returns the current theme for a given workspace. The purpose of this function is to determine which theme should be _set_. Open to other suggestions!

Reply:
Yes, that's right. How about `getUserTheme`? We have an upcoming ticket to move this preference from localStorage to user preferences, and that might work well for when we get there. If we wanted it to be super accurate for now I could also see `getLocalStorageTheme` or something.

Reply:
`getUserTheme` sounds good!